agent/rpm/pbench-agent.spec.j2

```diff
@@ -95,26 +95,35 @@ fi
 %posttrans
 
 %files
-%defattr(775,pbench,pbench,775)
+%defattr(664,root,root,775)
+/%{installdir}/ansible
 /%{installdir}/config
-
 /%{installdir}/lib
-/%{installdir}/ansible
-/%{installdir}/base
 
+/%{installdir}/base
 /%{installdir}/VERSION
 /%{installdir}/SEQNO
 /%{installdir}/SHA1
 /%{installdir}/profile
 
-%ghost %attr(0400,pbench,pbench) /%{installdir}/id_rsa
+%config(noreplace) %attr(0664,root,root) /%{installdir}/config/pbench-agent.cfg
 
-%defattr(775,pbench,pbench,775)
+%defattr(775,root,root,775)
```

Comment:
Why `775`?? Is there really anything here that's supposed to be writable?

Reply:
Do you mean, why not `555` (`-r-x`)?

Reply:
Yes; for directories that are executable scripts, `555` would seem appropriate. I don't see why the directories need `w` either?

Reply:
Only historical, as far as I can tell.